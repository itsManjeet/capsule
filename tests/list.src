println ([]);
println ([10, true, false, 10.034, 'c', "Hey"]);
println ([10] + [20]);
println (append([10, 20, 30], 20));
println (pop([10, 20, 30]));