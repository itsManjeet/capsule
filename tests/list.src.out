[]
[10, true, false, 10.034000, c, Hey]
[10, 20]
[10, 20, 30, 20]
[10, 20]
